ITE/6/README.adoc

```diff
@@ -36,58 +36,71 @@ endif::[]
-software artifacts. Instead of a fixed schema (`materials`, `products`,
-`command`, etc.), the attestation format allows users to define their own custom
-"predicates" with arbitrary schemas. This makes it easier and more natural to
-express any type of metadata about software. Several pre-defined predicate types
-cover the most common cases, including
-link:https://github.com/in-toto/attestation/blob/main/spec/predicates/provenance.md[provenance]
-for describing the origins of a software artifact,
-link:https://github.com/in-toto/attestation/blob/main/spec/predicates/spdx.md[spdx]
-for supporting link:https://spdx.dev[SPDX], and
-link:https://github.com/in-toto/attestation/blob/main/spec/predicates/link.md[link]
-as a drop-in replacement for the existing link schema.
-
-A future ITE will update the layout format to take advantage of this extra
-information.
-
-More broadly, this ITE moves in-toto to follow the
-link:https://slsa.dev/attestation-model[SLSA attestation model] and is developed
-jointly with
-link:https://cloud.google.com/binary-authorization[Binary Authorization]. The
-goal is to have an industry standard artifact metadata format that can be
-consumed by any system.
+This link:https://github.com/in-toto/ITE[in-toto enhancement (ITE)] introduces
+the in-toto Attestation Framework, a specification for generating and expressing
+verifiable claims about software supply chains. An attestation is authenticated
+metadata about software artifacts and the processes generating them. The
+framework allows users to define custom "predicates" with arbitrary,
+context-specific schemas. Several predicate types have been introduced that
+cover common cases, and accompanying ITEs have been proposed to define processes
+for creating attestation types and updating in-toto layouts to verify them.
 
 [[specification]]
 == Specification
 
-in-toto will support the new attestation format defined in
-<https://github.com/in-toto/attestation>. This defines the schema of the signed
-payload, which is `signed` in the original envelope and `payload` in
+NOTE: The
+link:https://github.com/in-toto/attestation[in-toto Attestation Framework] is
+defined in a stand-alone specification. This ITE presents a high level overview
+of the specification, and readers should refer to the latest versions of the
+full specification for all details.
+
+An attestation is an authenticated metadata that expresses one or more claims
+about software artifacts and the supply chain producing them. This means that as
+with traditional in-toto layout and link metadata, attestations are
+cryptographically signed and embedded in a signature envelope such as
 link:https://github.com/secure-systems-lab/dsse[DSSE].
 
+All attestations have the following general schema:
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/<VERSION>",
+  "subject": [
+    {
+      "name": "<NAME>",
+      "digest": {"<ALGORITHM>": "<HEX_VALUE>"}
```

Comment:
There is some discussion about possibly not using hex in a future version, or for other algorithms (e.g. go module dirsum). Maybe just `<VALUE>`?